src/main/java/uk/gov/hmcts/darts/common/repository/EventRepository.java

```diff
@@ -97,7 +97,7 @@ SELECT e.event_id, he.eve_id, string_agg(he.hea_id::varchar, ',' order by he.hea
         having count(e2.event_id) > 1
         limit :limit
         """, nativeQuery = true)
-    List<Integer> getCurrentEventIdsToBeProcessed(long limit);
+    List<Integer> findCurrentEventIdsWithDuplicates(long limit);
```

Comment:
Is it including duplicates or just those events that are duplicates?

Reply:
one eventId can have multiple duplicate events, so this is returning the eventIds with more than 1 event, not the ids of the events. eventId is not to be confused with the id of the event table (eventEntity.id), this is really badly named and does cause confusion.